data/coalition/coalition missions.txt

```diff
@@ -3111,3 +3111,129 @@ mission "Coalition: Outskirts Gauger Ready"
 				decline
+
+mission "Sativa: Struggling Author 1"
+	minor
+	source "Zug"
+	destination "Ki Patek Ka"
+	to offer
+		random < 30
+		credits > 140
+		has "Coalition: Contributor: done"
+	on offer
+		conversation
+			`	Upon entering the spaceport, you walk past a bookstore, currently holding a book signing event for an author known as 'Ann Oryza Sativa'. Nobody pays attention to her. It appears as if she is a struggling author.`
+			choice
+				`	(Ignore her.)`
+					decline
+				`	(Buy one of her books.)`
+					goto "buy her book"
+				`	(Sit down and talk to her.)`
+			label "talk to her"
+			`	You sit down on the table, next to Ann. She greets you with a smile. "Hello! Would you like to buy one of my books?" she asks.`
+			choice
+				`	"What are your books about?"`
+					goto "what are her books about"
+				`	"No, I'm not interested. How has life been as an author though?"`
+					goto "life as an author sucks"
+			label "what are her books about"
+			`	"I mostly write science fiction books. Today, I'm here to promote 'Beyond the Hyperdrive' a story in which a group of humans steal a bunch of jump drives from the Quarg. If I continue past this point I'd be spoiling it. Read it for yourself, you'll like it!"`
+			choice
+				`	"I'm interested. Wait here, I'll go buy myself a copy."`
+					goto "buy her book"
+				`	"Cool. How has life been lately as an author?"`
+			label "life as an author sucks"
+			`	"Bad," says Ann. "I've never had a bestseller or even a book that surpasses 10,000 copies sold."`
+			choice
+				`	(Tell her about the Coalition.)`
+					goto "tell her about the coalition"
+				`	"Well, that's gotta hurt. I hope you find success in your life one day.`
+					decline
+			label "buy her book"
+			action
+				payment -60
+			`	You enter the bookstore, ask the employee for one of Ann's books, and purchase it for 60 credits.`
+			choice
+				`	(Now go to Ann and get it signed.)`
+					goto "get it signed"
+				`	(Leave.)`
+					decline
+			label "get it signed"
+			`	You move to the table where Ann is sitting. She looks visibly happy as she grabs your book, takes out a pen, and signs it. "Thanks for buying one of my books. I don't get much sales and people like you mean the world to me."`
+			choice
+				`	"I appreciate it. How has life as an author been though?"`
+					goto "life as an author sucks"
+				`	"Thanks for signing my copy. I hope you find success in your life. Goodbye."`
+					decline
+			label "tell her about the coalition"
+			`	You check your surroundings, looking around to ensure you are not being listened to. Once you confirm that your conversation will be private, you begin speaking.`
+			`	"You know, I know of a place called the Coalition. They're a group of alien races. Maybe you could go there and find inspiration for your next novel, or sell your books there? I bet they'd love to see your writings!"`
```

Comment:
Usually the player doesn't talk in quotations like characters do. A regular narrative paragraph detailing the player explaining to Ann about the Coalition and suggesting her books there might be better.
Same for other instances further down on the missions.

Reply:
a3ff8fd06dc73ecb5ca957f617fe856807a18ecd